lib/resty/kafka/response.lua

```diff
@@ -15,12 +16,14 @@ local strbyte = string.byte
 local _M = {}
 local mt = { __index = _M }
 
+function _M.new(self, str, api_version)
+    api_version = api_version or request.API_VERSION_V0
```

Comment:
we should not use default value here, it may hide problems.
just remove this line will be better, that means the `api_version` is a required argument.

Reply:
updated. but that would be a breaking change

Reply:
Thanks :)
I think it won't be a breaking change since the users should not use the `response.new` directly?